### PR TITLE
Add unhelpful button

### PR DESCRIPTION
**Pull request by mruwnik**
image

One downside of this is that an article can be both liked and disliked at the same time. Though that can be a feature not a bug!

### CODE REVIEW

File: app/server-utils/stampy.ts [hunk 1]
```diff
@@ -452,3 +452,5 @@ export const incAnswerColumn = async (column: string, pageid: PageId, subtract:
 
 export const makeColumnIncrementer = (column: string) => (pageid: PageId, subtract: boolean) =>
   incAnswerColumn(column, pageid, subtract)
+export const makeColumnDecrementer = (column: string) => (pageid: PageId, subtract: boolean) =>
+  incAnswerColumn(column, pageid, !subtract)
```
Comment (Aprillion) [inline app/server-utils/stampy.ts:456]: eeh, `!subtract`?!? 😖

TBH, I would prefer to have 2 columns in Coda to track helpful and unhelpful (`2 - 1` has a different vibe from `100 - 99`), but if it's already decided that we want only 1 column please get rid of the `subtract` parameter from all places, replacing it with a `decAnswerColumn` function (or some better way)

File: app/routes/questions/actions.tsx
Comment (Aprillion) [inline app/routes/questions/actions.tsx:1]: 🤔 I am not a fan of the different button widths ... Figma from #285 does not show the button text, only the icons

=> @mruwnik I wonder if you could remove visible text and only use title for hover tooltips, please? or should we track in separate ~~Jira~~ GH issue?

@ccstan99 - Figma doesn't specify order for 3 buttons, is the order `👍👎✏️` going to be uncontroversial (and/or already approved from meetings) or do we want to discuss other options on Discord? (`👍✏️👎`, `✏️👍👎`, `✏️👎👍`, `👎👍✏️`, `👎✏️👍`)

**Review by Aprillion — CHANGES_REQUESTED** [review]

File: app/routes/questions/actions.tsx
Comment (mruwnik) [reply app/routes/questions/actions.tsx:1]: Separate issue, probably?

File: app/server-utils/stampy.ts (hunk 1 above)
Comment (mruwnik) [reply app/server-utils/stampy.ts]: There are now 2 columns

**Review by Aprillion — APPROVED** [review]

**Pull request merged**